src/uci.cpp

```diff
@@ -313,8 +313,8 @@ void UCIEngine::benchmark(std::istream& args) {
 
     Benchmark::BenchmarkSetup setup = Benchmark::setup_benchmark(args);
 
-    const int numGoCommands = count_if(setup.commands.begin(), setup.commands.end(),
-                                       [](const std::string& s) { return s.find("go ") == 0; });
+    const long long numGoCommands = count_if(setup.commands.begin(), setup.commands.end(),
```

Comment:
maybe just `auto` or `ptrdiff_t`? This doesn't make sense on 32-bit

Reply:
ptrdiff_t required an extra header.  I have a pet peeve against auto because i don't like looking at code and having to do extra work just to find out somethings type.